src/main/webapp/usermap.jsp

```diff
@@ -15,7 +15,7 @@
     <h1 class="caption">Who is using TEAMMATES?</h1>
     <div id="contentHolder">
         <div id="container" style="position: relative; width: 800px; height: 500px; border: 1px solid #DEDEDE;"></div>
-        <p id="lastUpdate" class="lastUpdate">Last updated: 30 November 2016</p>
+        <p id="lastUpdate" class="lastUpdate">Last updated: <span id="lastUpdateDate" class="lastUpdate"></span></p>
```

Comment:
![screenshot 50](https://cloud.githubusercontent.com/assets/12005551/22078307/8e746e04-dddd-11e6-85f5-0e075e239d74.png)
 after the removla of id and class to p tag